File: math/Go/bench.go
package main

import "fmt"

func compute(number int) {
	i := 0
	for i < number {
		if i % 123 == 0 {
			i = i + 10
		} else {
			i = i + 1
		}
	}
	fmt.Println("result is", i)
}

func main() {
	compute(1000000000)
}

File: math/Javascript/bench.js
function compute(number) {
	var i = 0;
	while (i < number) {
		if (i % 123 == 0) {
			i = i + 10;
		} else {
			i = i + 1;
		}
	}
	console.log("result is " + i);
}

compute(1000000000);
